### PR TITLE
Changing the proton propagator to the Optical Function method

### PR DESCRIPTION
**Pull request by mundim**
#### PR description:

Simple change in the configuration file to use the approximate optical function for Run3 instead of the HECTOR (which uses ideal optics from LHC). It needs the record LHCInfoRcd and CTPPSOpticsRcd, which are in process of being integrated into the GT
#### PR validation:
All standard test has been done: scram b runtests, scram b code-checks and scram b code-formats

The runTheMatrix  -l limited -i all --ibeos ran without any error with the official GT, but in this case, since the record below is not in any official GT, an ESSource has been provided on config. 

Then the essource was removed and runTheMatrix was tested with the GT 113X_mcRun3_2021_realistic_Candidate_2021_04_06_19_59_53 with no error, however, the correct GT should be 113X_mcRun3_2021_realistic_Candidate_2021_04_09_15_03_03 (14TeV scenario), in which case, there was some wf that failed, however, the wf that runs PPS works fine (the GT above was proposed to solve the problem of #33266 )

In summary: this PR needs the GT 113X_mcRun3_2021_realistic_Candidate_2021_04_09_15_03_03 to be integrated.

### CODE REVIEW

**Comment by civanch** [comment]
please test

**Comment by mundim** [comment]
This looks strange to me LHCInfoRcd tag is indeed included in the GT candidate I mentioned above as well as in the last official GT (apparently, according got the dbcond browser). I tested right now the wf 11634.0,12434.0 and it worked, but not the 11634.911 with the same message of missing LHCInfoRcd source. I ran the code locally and it indeed finds the record with no problem.

**Comment by mundim** [comment]
Hello
I checked again using the latest IB (CMSSW_11_3_X_2021-04-15-1100) in which all needed tags are included, and all runTheMatrix tests ran without any problem. Is is possible to have this PR integrated into CMSSW_11_3_0?
Thanks

**Comment by civanch** [comment]
please test

**Comment by civanch** [comment]
@mundim , this PR will be tested against master - 12_0_X now. If will be accepted it will be possible to discuss backport to 11_3_0.

**Comment by mundim** [comment]
> @mundim , this PR will be tested against master - 12_0_X now. If will be accepted it will be possible to discuss backport to 11_3_0.

@civanch, yes, that would be great. Is it automatically queued for back port or do I have to submit a special request?

**Comment by civanch** [comment]
+1

**Comment by civanch** [comment]
@mundim , you need making a separate PR to 11_3_X branch - you may copy files from this PR.

**Comment by mundim** [comment]
> @mundim , you need making a separate PR to 11_3_X branch - you may copy files from this PR.

Hi @civanch, I'm afraid I'm not sure about what this means, since this branch was created under 11_3_X one.

**Comment by silviodonato** [comment]
please test workflows 11725.0, 11925.0

**Comment by silviodonato** [comment]
cc @cms-sw/ctpps-dpg-l2

**Comment by silviodonato** [comment]
+1
